# Gemini335 depth image data
int32 width            # Depth image width (640)
int32 height           # Depth image height (480)
uint16[] depth_data    # Depth data array (width*height elements)
uint64 timestamp       # Timestamp in milliseconds